### PR TITLE
Fixed Meeting Token Sample Issues

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, approving!

**Review by Prithvi-MSFT — APPROVED** [review]

File: samples/meetings-token-app/csharp/Service/MeetingService.cs [hunk 1]
```diff
@@ -33,18 +34,18 @@ public class MeetingService : IMeetingService
...
         /// <param name="botCredentials">The Bot credentials.</param>
-        /// <param name="botAdapter">The Bot Framework HTTP adapter.</param>
+        /// <param name="adapters">The Bot Framework HTTP adapter.</param>
         /// <param name="tenantInfoRepository">The TenantInfo repository.</param>
         /// <param name="meetingServiceOptions">App options related to the meetings service.</param>
         public MeetingService(
             IHttpClientFactory httpClientFactory,
             AppCredentials botCredentials,
-            IBotFrameworkHttpAdapter botAdapter,
+            IBotFrameworkHttpAdapter adapters,
```
Comment (ChetanSharma-msft) [inline samples/meetings-token-app/csharp/Service/MeetingService.cs:43]: It's a single adapter, please rename it like `botAdapter` or `botFrameworkHttpAdapter`

File: samples/meetings-token-app/csharp/Service/MeetingService.cs [hunk 2]
```diff
@@ -54,10 +55,15 @@ public class MeetingService : IMeetingService
         /// <inheritdoc/>
         public async Task<UserMeetingRoleServiceResponse> GetMeetingRoleAsync(string meetingId, string userId, string tenantId)
         {
-            var serviceUri = this.tenantInfoRepository.GetServiceUrl(tenantId);
-            if (serviceUri == null)
+            var serviceUri = string.Empty;
+
+            if (this.tenantInfoRepository.GetServiceUrl(tenantId) != null)
```
Comment (ChetanSharma-msft) [inline samples/meetings-token-app/csharp/Service/MeetingService.cs:60]: Why do we need to call this method twice?
this.tenantInfoRepository.GetServiceUrl(tenantId)

We can assign it's value in a variable and can check for null?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments on this PR

File: samples/meetings-token-app/csharp/Service/MeetingService.cs (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/meetings-token-app/csharp/Service/MeetingService.cs:43]: Fixed

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**